usage-collection/src/main/proto/usage.proto

```diff
@@ -164,6 +164,8 @@ message Event {
     CDPVMDiagnosticsEvent cdpVmDiagnosticsEvent = 76;
     // A CDP DataWarehouse Virtual Warehouse's nodes changed.
     CDPCDWVirtualWarehouseNodesChanged cdpCDWVirtualWarehouseNodesChanged = 77;
+    // A CDP stack patcher event fired.
+    CDPStackPatchEvent cdpStackPatchEvent = 79;
```

Comment:
Why don't you use the next number right after cdpCDWVirtualWarehouseNodesChanged (78)?